notebooks/simple-feature-transformations-explain-local.ipynb

```diff
@@ -0,0 +1,512 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# 1. Using SHAP TabularExplainer\n",
+    "# clf.steps[-1][1] returns the trained classification model\n",
+    "explainer = TabularExplainer(clf.steps[-1][1], \n",
+    "                             initialization_examples=x_train, \n",
+    "                             features=attritionXData.columns, \n",
+    "                             classes=[\"Not leaving\", \"leaving\"], \n",
+    "                             transformations=transformations)\n",
+    "\n",
+    "\n",
+    "\n",
+    "\n",
+    "# 2. Using MimicExplainer\n",
+    "# augment_data is optional and if true, oversamples the initialization examples to improve surrogate model accuracy to fit original model.  Useful for high-dimensional data where the number of rows is less than the number of columns. \n",
+    "# max_num_of_augmentations is optional and defines max number of times we can increase the input data size.\n",
+    "# LGBMExplainableModel can be replaced with LinearExplainableModel, SGDExplainableModel, or DecisionTreeExplainableModel\n",
+    "# explainer = MimicExplainer(clf.steps[-1][1], \n",
+    "#                            x_train, \n",
+    "#                            LGBMExplainableModel, \n",
+    "#                            augment_data=True, \n",
+    "#                            max_num_of_augmentations=10, \n",
+    "#                            features=attritionXData.columns, \n",
+    "#                            classes=[\"Not leaving\", \"leaving\"], \n",
+    "#                            transformations=transformations)\n",
+    "\n",
+    "\n",
+    "\n",
+    "\n",
+    "\n",
+    "# 3. Using PFIExplainer\n",
+    "\n",
+    "# Use the parameter \"metric\" to pass a metric name or function to evaluate the permutation. \n",
+    "# Note that if a metric function is provided a higher value must be better.\n",
+    "# Otherwise, take the negative of the function or set the parameter \"is_error_metric\" to True.\n",
+    "# Default metrics: \n",
+    "# F1 Score for binary classification, F1 Score with micro average for multiclass classification and\n",
+    "# Mean absolute error for regression\n",
+    "\n",
+    "# explainer = PFIExplainer(clf.steps[-1][1], \n",
+    "#                          features=x_train.columns, \n",
+    "#                          transformations=transformations,\n",
+    "#                          classes=[\"Not leaving\", \"leaving\"])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Generate global explanations\n",
+    "Explain overall model predictions (global explanation)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Passing in test dataset for evaluation examples - note it must be a representative sample of the original data\n",
```

Comment:
@imatiach-msft do you think we should change the second half of this line a bit? This seems to me that it might imply that evaluation data has to be a sample of initialization data?

Reply:
Maybe it's "original data" that is vague to me

Reply:
That was the original. I'll merge and we can ask @imatiach-msft for further insight.